cmd/abc/abc.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2023 The Authors (see AUTHORS file)
+// Copyright 2024 The Authors (see AUTHORS file)
```

Comment:
Actually, don't update this. See go/copyright, but you only change the copyright for a totally _new_ file.

Reply:
done